docker/regtest/regtest-control.sh

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+trap cleanup SIGINT SIGTERM ERR EXIT
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+
+usage() {
+  cat <<EOF
+Usage: $(basename "${BASH_SOURCE[0]}") [-h] [-v] [-f] -p password arg1 [arg2...]
```

Comment:
No there are no args (yet).
I do not know if we want multiple scripts for any upcoming task/automation, or if it is better to have a single script that will take args in the future, hence the name `regtest-control` instead of `fund-regtest-wallet` (or similar).

I guess I should rename the file and we'll see if we need a "management" script in the future.

What do you suggest? What do you think is better?
I will of course delete the unused code even before the decision.